llvm/lib/Transforms/IPO/LowerTypeTests.cpp

```diff
@@ -470,6 +470,9 @@ class LowerTypeTestsModule {
 
   Function *WeakInitializerFn = nullptr;
 
+  GlobalVariable *GlobalAnnotation;
+  std::set<void *> FunctionAnnotations;
```

Comment:
Use SmallPtrSet or DenseSet, not std::set. Also, why does this use `void *` pointers? Shouldn't this be something like `GlobalValue *` or at the very least `Value *`?

Reply:
There are quite some uses of `std::set` in this source file, and only one`DenseSet`, so I just selected `std::set`. I can change to use `DenseSet`.

With `void *` I can simply not to worry about casting between types, especially when to query the set to see if it is function annotation.